### ISSUE TITLE
cant connect to server over ssh

### ISSUE THREAD

**Issue by NicholasBHubbard** [NicholasBHubbard/yabsm#3]
This is just a small word on something you may be confused by. All of this is explained in the manual, but I don't think it hurts to reiterate.

You may see this error either in logs (/var/log/yabsm) or when running yabsm interactively:
```
yabsm: ssh error: $SSH_BACKUP_NAME: cannot establish SSH connection: unable to establish master SSH connection: master process exited unexpectedly
```
This means that the "yabsm" user on your system cannot connect to your ssh_backup's SSH server using key-based authentication.

Remember, yabsm works by creating a locked user on your host system named "yabsm". This user is given a key pair that should be used to log onto servers with SSH. This public key can be seen by running the following command (as root): 
```
# yabsm c ssh-key
```
Now, from the server you are ssh'ing into (all the details of which are determined by a `ssh_backup` you defined in your config), append this public key to the server users `$HOME/.ssh/authorized_keys` file.

Now, to make sure everything is setup correctly, from your yabsm host run this command (as root):
```
# yabsm c ssh-check $SSH_BACKUP_NAME
```
If everything is setup then this will just print "all good". Otherwise it will display a (hopefully useful) error message about what is wrong.

 As a last resort, sign in as the yabsm user and try to ssh into the ssh_backups `ssh_dest`. 
```
# su yabsm
$ ssh -o PreferredAuthentications=publickey -o PasswordAuthentication=no fred@192.168.1.73
```
Bottom line is if you can sign into the `ssh_dest` as the "yabsm" user using passwordless key-based authentication, then yabsm should just work.
